### ISSUE TITLE
Learning rate scheduler

### ISSUE THREAD

**Issue by WenjieDu** [WenjieDu/PyPOTS#194]
### 1. Feature description

Add learning rate schedulers to enhance model optimization.

### 2. Motivation

Models like Transformer may need pretraining or warm-up.

### 3. Your contribution

Will create PR to make this happen.
